### PR TITLE
fix: setup single db instance

### PR DESCRIPTION
**Pull request by spaenleh**

### CODE REVIEW

File: .devcontainer/docker-compose.yml [hunk 1]
```diff
@@ -201,6 +197,16 @@ services:
...
+  # nginx:
+  #   container_name: reverse_proxy
+  #   image: nginx:1-alpine
+  #   volumes:
+  #     - ./nginx-templates:/etc/nginx/templates
+  #   environment:
+  #     - NGINX_HOST=localhost
+  #     - NGINX_PORT=3000
+  #   ports:
+  #     - 3000:3000
```
Comment (spaenleh) [inline .devcontainer/docker-compose.yml:209]: To remove

File: README.md [hunk 2]
```diff
@@ -46,16 +46,14 @@ First open the folder in the dev-container by using the command palette <kbd>cmd
...
 - `graasp-core` : Node.js backend of Graasp
-- `graasp-postgres` : PostgreSQL database used by the backend
-- `graasp-postgres-test` : PostgreSQL database used when running tests
+- `db` : PostgreSQL cluster used by multiple services
 - `graasp-etherpad` : Container for the etherpad service
-- `graasp-postgres-etherpad` : PostgreSQL database used by the etherpad service
 - `graasp-meilisearch` : Container for the meilisearch service
 - `graasp-redis` : Redis instance to enable websockets
 - `graasp-localstack` : Localstack instance use to fake S3 storage locally
+- `localfile` : Simple static file server to get files stored in graasp when using the `local` storage otpion (see the [Utilities section](#utilities))
```
Comment (pyphilia) [inline README.md:54]: `option`

File: .devcontainer/docker-compose.yml [hunk 3]
```diff
@@ -181,7 +177,7 @@ services:
     container_name: graasp-iframely
     image: graasp/iframely:latest
     environment:
-      NODE_ENV: staging # why "staging" ??
+      NODE_ENV: production # required in order for the responses to not timeout ???
```
Comment (pyphilia) [inline .devcontainer/docker-compose.yml:180]: Remove `???` ???

File: .devcontainer/docker-compose.yml [hunk 4]
```diff
@@ -147,6 +124,25 @@ services:
     volumes:
       - ./meilisearch_data:/meili_data
 
+  umami:
+    container_name: umami
+    image: ghcr.io/umami-software/umami:postgresql-latest
```
Comment (pyphilia) [inline .devcontainer/docker-compose.yml:129]: Should we set the postgres version? Since we've set 15 for the local db.

**Review by pyphilia — APPROVED** [review]
LGTM, just some questions.

File: .devcontainer/docker-compose.yml (hunk 4 above)
Comment (spaenleh) [reply .devcontainer/docker-compose.yml:129]: This is the tag defined by umami, it does not relate to the version of postgres, basically they have a postgres and a mysql version for docker, and the tags are `postgres-latest` and `mysql-latest`

**Pull request merged**